### ISSUE TITLE
How to choose Cluster Diameters?

### ISSUE THREAD

**Issue by darjaved** [prisms-center/CASMcode#353]
What is the best way to choose the clusters diameters when we have the different size supercells?

**Comment by xivh** [comment]
If you plot the coefficients of your model normalized by the number of sites in the cluster, you should see a decay in magnitude with respect to number of sites and the cluster max length.

**Comment by darjaved** [comment]
Yeah i got that, can i choose the maximum cluster size as the maximum sphere that can be fixed in my supercell

e.g., i have a supercell with dimensions of 5, 6, 10 angstroms, so the maximum diameter will be 5 angstroms. If i choose the higher values that may lead to the undesirable interaction between the images. 

In the CASM we usually have supercells of different sizes, so i have to select diameters based on the largest size supercell?

**Comment by darjaved** [comment]
Also, is it ok to get all the quadruplets of almost equal size (radii) with different ECI's? I may get quadruplets of different size at higher value of cutoff but that will not improve my model i guess.

**Comment by xivh** [comment]
The cluster size does not have to be smaller than the supercell size. It is okay if they cross between supercells.

For the second question, yes it is okay. There could be some strong interactions that occur just for certain clusters. For example, you could have a tetrahedron and a square, and maybe one is more influential than the other.
